helix-term/src/keymap.rs

```diff
@@ -80,6 +80,7 @@ impl Default for Keymaps {
             // and matching set for select mode (extend)
             //
             key!('r') => Command::replace,
+            key!('~') => Command::change_case,
```

Comment:
Can you place this somewhere else? It's between `r` and `R` which are a pair.

Reply:
Done